Sf rest lx warning (#60)
skuid/skuid-labs: experiments/dataSources/SalesforceREST/readme.md

####  Query contacts   
* Query method with this URL:  ``/services/data/v46.0/query/?q=SELECT+id,FullName,Email+from+Contact``
* The query string accepts your full SOQL query.     

## Deploy this data source in Lightning Experience

### A warning!

> Salesforce does not allow a custom lightning component to access their own Rest API. For more details [look here.](https://salesforce.stackexchange.com/questions/55306/how-to-call-a-salesforce-rest-url-from-lightning-component) 

Reviewer: ```suggestion
Salesforce does not allow custom Lightning components to access their own REST API. For more details [see this StackExchange post.](https://salesforce.stackexchange.com/questions/55306/how-to-call-a-salesforce-rest-url-from-lightning-component) 
```

> * If you try to deploy a Skuid pages built with these Rest models using the Skuid Page Lightning component - it won't work. You will just get CORS errors. You will get no Data. (Boo)  

Reviewer: ```suggestion
* If you try to deploy a Skuid pages built with these REST models using the Skuid Page Lightning component, it won't work. You will just get CORS errors. You will get no data. (Boo)  
```
